runtime/autoload/man.vim

```diff
@@ -112,7 +106,7 @@ endfunction
 
 function! s:get_path(sect, name) abort
   if empty(a:sect)
-    let path = system(s:man_cmd.' '.s:man_find_arg.' '.shellescape(a:name))
+    let path = system([s:man_cmd, s:man_find_arg, a:name])
```

Comment:
@nhooyr Thanks. So @ooora we need to find some other way to make tcsh happy in the cases where man.vim uses `s:man_cmd`.